### DISCUSSION TITLE
Customization

### DISCUSSION THREAD

**Discussion by Jijiri89** [filamentphp/filament#5941] [General]
Plesaes how do I connect the filament login page to the default Laravel welcome.blade.php and Jetstream AUTH?

**Comment by danharrin** [comment]
theyre separate things, why do you need that? if you need full authentication, try the Filament Breezy plugin

**Reply by Jijiri1** [reply]
OK. With that plugin gives me ability to have a full stack without using blade or inertia stack in laravel? I am really enjoying filament.

I want the way to make the authenticate and a complete home pay using filament

**Reply by danharrin** [reply]
Breezy uses Blade and Livewire the same as Filament does

**Comment by Jijiri89** [comment]
Great, thank you. I really love filament but I am a bit new. What I
actually want is to be able to have frontend which I can style without
using blade or inertia stack.

**Comment by Jijiri89** [comment]
Ok, thanks.
Please, also is there any way to customize tables that display view in the
filament admin panel? Let say if I want to apply borders to the table.

**Reply by danharrin** [reply]
Using CSS

**Comment by Jijiri89** [comment]
Thanks, where exactly can I edit to apply CSS?

**Reply by danharrin** [reply]
In your admin panel theme CSS file

**Comment by Jijiri89** [comment]
Thanks,  In the config directory? I can't find it

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/admin/appearance#building-themes

**Comment by Jijiri89** [comment]
Got it, thanks!

**Comment by Jijiri89** [comment]
Now, how do I create the links or navigation links in my home page to point
to the breezy Auth?

**Comment by Jijiri1** [comment]
OK, thank you, yesterday I deleted the default content of the welcome.
Blade. Php, I think that is why I am seeing the breezy Auth links in the
welcome page. In this case, how do you add the link to breezy Auth to the
welcome.blade?

**Comment by Jijiri89** [comment]
Please, is it advisable to use bootstrap in the frontend as filament uses
tailwindcss?

**Comment by Jijiri89** [comment]
Please what is the CSS class name for the tables? I want to apply CSS to it
but I don't know there class names
